### ISSUE TITLE
how to train controlnet with 2 control images?

### ISSUE THREAD

**Issue by XiangjiBU** [lllyasviel/ControlNet#536]
how to train controlnet with 2 control images?

**Comment by geroldmeisinger** [comment]
duplicate: https://github.com/lllyasviel/ControlNet/discussions/30

**Comment by XiangjiBU** [comment]
> duplicate: #30

thanks, but I am not asking multi-controlnet, I want to know how to add another input channel and train a controlnet with 2 control images

**Comment by geroldmeisinger** [comment]
yes, this is exactly what this thread is about, see here which makes it more clear https://github.com/lllyasviel/ControlNet/discussions/30#discussioncomment-7042847
for additional channel, all info I could find is https://github.com/lllyasviel/ControlNet/issues/271
